tests/ser_de.rs

```diff
@@ -1,7 +1,11 @@
+extern crate bitcoin;
 extern crate bitcoin_hashes;
+extern crate hex;
 extern crate lightning_invoice;
 extern crate secp256k1;
 
+use bitcoin::secp256k1::Secp256k1;
```

Comment:
You could write `use secp256k1::Secp256k1` directly since it's already a dependency, which would make `bitcoin` redundant as a dev-dep.